src/inputs/labeled-inputs.scss

```diff
@@ -61,8 +61,8 @@
       color: t(iui-text-color-muted);
     }
 
-    > svg,
-    :not(.iui-button) svg {
+    > .iui-icon,
+    :not(.iui-button) .iui-icon {
```

Comment:
I think you could revert this. We can target all svgs under `.iui-message`

Reply:
Reverted.

Reply:
You have reintroduced `> svg`.

When I said "target all svgs under .iui-message", I meant just `svg`.